blogs/whats_new_1.6/index.md

```diff
@@ -0,0 +1,225 @@
+    When a flow is a duplicate on another interface, instead of saving multiple copies of that flow data, only one is stored now.  It keeps track of the list of interfaces that refer to the flow data.  This greatly reduces the amount of storage.
+
+    The raw JSON fields have been renamed and pluralized to `Interfaces` and `IfDirections`, since the type changed to a list from a single value.  Note the use of square brackets for the lists below.
+
+    ```
+      "IfDirections": [
+        0
+      ],
+      "Interfaces": [
+        "br-ex"
+      ],
+    ```
+
+3. Subnet labels<br>
+    You can define a list of names for CIDRs (IP ranges and/or hosts).  When the CIDR matches, the name appears in the flow data as the `SrcSubnetLabel` or `DestSubnetLabel` field.
+
+    YAML Example:
+
+    ```
+      processor:
+        subnetLabels:
+          customLabels:
+          - cidrs:
+            - 10.129.0.73/32  # provide a name for this host
+            name: querier
+          openShiftAutoDetect: false
+    ```
+
+    If you don't want to define names, you can enable the OpenShift auto-detect mode to identify the items as "Pods", "Services", "Machines" or "n/a" (not applicable).  You configure this in the Processor section of Flow Collector (Figure 8).
+
+![Subnet labels configuration](images/subnet_labels_configuration.png)
+    _<div style="text-align: center">Figure 8: Subnet labels configuration</div>_
+
+   Figure 9 shows the subnet label columns in the Traffic flow table when OpenShift auto-detect is enabled.
+
+![Subnet labels columns](images/subnet_labels_columns.png)
+    _<div style="text-align: center">Figure 9: Subnet labels columns</div>_
+
+
+## Custom Metrics
+
+In my [Network Observability 1.5 blog](https://developers.redhat.com/articles/2024/03/20/whats-new-network-observability-15), I covered the development preview of the FlowMetrics API.  While it's possible to take any [flow data labels](https://docs.openshift.com/container-platform/4.16/network_observability/json-flows-format-reference.html) and turn it into a Prometheus metric, that web page was updated to indicate which fields can be safely used as labels to avoid high cardinality.
+
+This feature is now GA, and includes more filtering options (see [reference](https://github.com/netobserv/network-observability-operator/blob/main/docs/FlowMetric.md)) and support for [dashboards in OpenShift](https://github.com/netobserv/network-observability-operator/blob/main/docs/Metrics.md#charts-openshift-only).  That link also has many more examples of using custom metrics.
+
+
+## Network Observability CLI
+
+This release has yet another significant technical preview feature called Network Observability CLI (NetObserv CLI).  It is a kubectl plug-in that effectively installs Network Observability from the command line but is text-based only.  When you exit the program, it cleans up and removes itself.  In other words, you can now run Network Observability on-demand!
```

Comment:
> It is a kubectl plug-in that effectively installs Network Observability from the command line but is text-based only.

You may add it's only some netobserv components but not all / not the operator? People may think it's an alternative way of installing the operator